config.yml

```diff
@@ -7723,3 +7737,2033 @@ production_recipes:
+    name: Create Leather Chestplate
+    production_time: 40
+    inputs:
+      Leather Chestplate:
+        material: LEATHER_CHESTPLATE
+        amount: 1
+      Emerald Blocks:
+        material: EMERALD_BLOCK
+        amount: 1
+    outputs:
+      Enchanted Leather Chestplate:
+        material: LEATHER_CHESTPLATE
+        amount: 1
+    enchantments:
+      Unbreaking 3:
+        type: DURABILITY
+        level: 3
+        probability: 0.04
+      Protection 3:
+        type: PROTECTION_ENVIRONMENTAL
+        level: 3
+        probability: 0.04
+      Projectile Protection 4:
+        type: PROTECTION_PROJECTILE
+        level: 4
+        probability: 0.27
+      Protection 4:
+        type: PROTECTION_ENVIRONMENTAL
+        level: 4
+        probability: 0.03
+      Thorns 2:
+        type: THORNS
+        level: 2
+        probability: 0.19
+      Fire Protection 3:
+        type: PROTECTION_FIRE
+        level: 3
+        probability: 0.18
+      Fire Protection 4:
+        type: PROTECTION_FIRE
+        level: 4
+        probability: 0.12
+      Blast Protection 4:
+        type: PROTECTION_EXPLOSIONS
+        level: 4
+        probability: 0.12
+      Thorn 1:
+        type: THORNS
+        level: 1
+        probability: 0.01
+  Create_Leather_Helmet:
+    name: Create Leather Helmet
+    production_time: 40
+    inputs:
+      Leather Helmet:
+        material: LEATHER_HELMET
+        amount: 1
+      Emerald Blocks:
+        material: EMERALD_BLOCK
+        amount: 1
+    outputs:
+      Enchanted Leather Helmet:
+        material: LEATHER_HELMET
+        amount: 1
+    enchantments:
+      Unbreaking 3:
+        type: DURABILITY
+        level: 3
+        probability: 0.06
+      Protection 3:
+        type: PROTECTION_ENVIRONMENTAL
+        level: 3
+        probability: 0.04
+      Projectile Protection 4:
+        type: PROTECTION_PROJECTILE
+        level: 4
+        probability: 0.19
+      Protection 4:
+        type: PROTECTION_ENVIRONMENTAL
+        level: 4
+        probability: 0.03
+      Aqua Affinity 1:
+        type: WATER_WORKER
+        level: 1
+        probability: 0.18
+      Respiration 3:
+        type: OXYGEN
+        level: 3
+        probability: 0.17
+      Fire Protection 3:
+        type: PROTECTION_FIRE
+        level: 3
+        probability: 0.13
+      Fire Protection 4:
+        type: PROTECTION_FIRE
+        level: 4
+        probability: 0.10
+      Blast Protection 4:
+        type: PROTECTION_EXPLOSIONS
+        level: 4
+        probability: 0.10
+      Respiration 2:
+        type: OXYGEN
+        level: 2
+        probability: 0.01
+  Wood_Plains_Crate:
+    name: Package Plains Wooden Crate
+    production_time: 60
+    inputs:
+      Wheat:
+        material: WHEAT
+        amount: 512
+      Raw Potato:
+        material: POTATO
+        amount: 940
+      Dirt:
+        material: DIRT
+        amount: 384
+    outputs:
+      Wood Crate:
+        material: STORAGE_MINECART

```

Comment:
Is storage minecart a good choice here? Seems risky, easy to place while travelling to destination causing loss of lore.